whiteblock/cmd/transactions.go

```diff
@@ -37,9 +38,28 @@ The primary use of these methods is to be able to send one line commands through
 testing script that will be able to automate transaction tests.
 */
 
+var sendTxCmd = &cobra.Command{
```

Comment:
`whiteblock tx send` should be the command to send a transaction. Adding the addition word single makes it too verbose. Remove this, as we don't want it as the base.